### ISSUE TITLE
French colonies in India

### ISSUE THREAD

**Issue by settintotrieste** [settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE#149]
Now that we have added the smaller Portuguese colonies in India, perhaps we should also add the French smaller colonies in India as well for consistency? Karikal and Yanaon on the Coromandel Coast, Mahé on the Malabar Coast and Chandernagor in Bengal can be added as well in a similar fashion as the Portuguese Indian colonies by making them enclaves of Pondicherry. This is historically accurate as well as these small colonies were under an administrative union with Pondicherry.

**Comment by ARR8** [comment]
Gladly, but I would like to know what you would like me to do with their populations. For the Portuguese colonies, I added them to Goa, but I wasn't sure if I should have reduced any from the provinces I took pixels off of. I'm not too worried about it since India is slightly underpopulated now after #133, but it'd still be nice to know.

**Comment by settintotrieste** [comment]
Sure, you can do the same as with Goa and add the population to Pondicherry without removing pops from the affected provinces. The under population of India should compensate for the extra pops from the French colonies.

**Comment by settintotrieste** [comment]
![invasion-of-portugal-1024x933](https://cloud.githubusercontent.com/assets/24409479/25658217/5b760cc0-3034-11e7-90b1-9a8a0ca81e05.gif)

Based on this map, only the colonies of Diu, Karikal, Goa and Pondicherry are really visible on the map so I think we should correct the current setup to only display these colonies on the map.

**Comment by ARR8** [comment]
I'm of the opinion we should add every coastal colony to allow naval docking in case we ever change the sea tiles

**Comment by settintotrieste** [comment]
This is something that we can do though I'll prefer to keep this on hold first. We can see revisit this again for 1.07.
